draft-ietf-quic-transport.md

```diff
@@ -2152,35 +2155,41 @@ receive packets without first having sent a packet on that path. Effective NAT
 traversal needs additional synchronization mechanisms that are not provided
 here.
 
-An endpoint MAY include PATH_CHALLENGE and PATH_RESPONSE frames that are used
-for path validation with other frames.  In particular, an endpoint can pad a
-packet carrying a PATH_CHALLENGE for Path Maximum Transfer Unit (PMTU)
-discovery (see {{pmtud}}), or an endpoint can include a PATH_RESPONSE with its
-own PATH_CHALLENGE.
+An endpoint MAY include other frames with the PATH_CHALLENGE and PATH_RESPONSE
+frames used for path validation.  In particular, an endpoint can include
+PADDING with a PATH_CHALLENGE for Path Maximum Transfer Unit (PMTU) discovery
+(see {{pmtud}}), and/or include a PATH_CHALLENGE with its own PATH_RESPONSE.
 
 When probing a new path, an endpoint might want to ensure that its peer has an
 unused connection ID available for responses. The endpoint can send
 NEW_CONNECTION_ID and PATH_CHALLENGE frames in the same packet. This ensures
 that an unused connection ID will be available to the peer when sending a
 response.
 
+The number of new paths an endpoint can probe is limited by the number of
+extra Connection IDs the peer has previously supplied. Supplying a peer with
+multiple Connection IDs allows it to simultaneously probe multiple paths.
```

Comment:
It's important to note that multiple paths may be probed, but I don't think we want to imply that this is the only reason to supply a peer with multiple connection IDs. How about something similar to
```suggestion
An endpoint can choose to simultaneously probe multiple paths. The number
of simultaneous paths used for probes is limited by the number of extra 
Connection IDs its peer has previously supplied, since each new local address 
used for a probe requires a previously unused Connection ID.
```